src/main/java/dev/flab/studytogether/domain/member/service/MemberV2Service.java

```diff
@@ -53,4 +55,29 @@ public MemberV2 signUp(String email, String password, String nickname) {
 
         return newMember;
     }
+
+    @Transactional
+    public void emailAddressAuthenticate(String email, String authKey) {
+        MemberV2 member = memberV2Repository.findByEmail(email)
+                .orElseThrow(LinkNotValidException::new);
```

Comment:
email에 해당하는 멤버를 찾는데 없으면 링크가 유효성 예외가 뜬다는 조금 어색하지 않나요?

무언갈 찾았는데 없으면 없는 주체에 대해서 표현해주는 게 더 명확할 것 같아요.
```suggestion
        MemberV2 member = memberV2Repository.findByEmail(email)
                .orElseThrow(MemberNotFoundException::new);
```

Reply:
반영 완료 [7282926](https://github.com/wooni97/StudyTogether/pull/5/commits/728292653c910990b652ec8ec5ef4b581f83b0c7)